app/views/orders/new.html.erb

```diff
@@ -3,6 +3,48 @@
   <%= form_for(@order) do |f| %>
```

Comment:
The html for this form is a little verbose. You can simplify it a bit by using [simple_form](https://github.com/plataformatec/simple_form). It'll produce the same html, but require less code.